INSTALL-SharedToken.md

```diff
@@ -0,0 +1,192 @@
+aaf-shibext
```

Comment:
maybe rename this file to `README.md` so that it will be rendered when viewing this repo, rather than having to click into it